build.properties

```diff
@@ -8,4 +8,5 @@ mappings=20201028-1.16.3
 # Dependencies
 jei_version=7.6.1.71
 structurize_version=0.13.148-ALPHA
+datagen_version=0.1.48-ALPHA
```

Comment:
Note this version doesn't exist yet.  It's specified in anticipation of ldtteam/DataGenerators#2 being merged.

Reply:
Just merged, should be available soon